seed/seed16.go

```diff
@@ -112,6 +112,19 @@ func (s *seed16) Model() (*asserts.Model, error) {
 	return s.model, nil
 }
 
+func (s *seed16) Brand() (*asserts.Account, error) {
```

Comment:
this could share a very simple helper in helpers.go:

findBrand(seed Seed, db asserts.RODatabase) (*asserts.Account, error)

inside it can use seed.Model()